### PR TITLE
Announce Sourcegraph 3.9

### PR DESCRIPTION
**Pull request by sqs**

### CODE REVIEW

**Comment by sqs** [comment]
Merging when next deploy is green. Will address https://github.com/sourcegraph/about/issues/287 later.

**Comment by slimsag** [comment]
I pushed slight change to paginated search docs link -- sorry about that being wrong earlier!

**Comment by sqs** [comment]
@slimsag I actually intended to link to the tech architecture doc. It's an experimental mention, so the audience is naturally more technical, and the doc is a great read for anyone interested in this topic (that also enhances our credibility). Your change now links to the same doc page 2x. Any reason you don't want to link to the architecture doc?

File: blogposts/announcing-sourcegraph-3.9.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+## Export settings for easier bug reporting
+
+Sourcegraph settings and configuration can now be viewed on a single page for easier exporting when reporting a bug. To view, go to ‘Report a bug’ on the site admin side panel or navigate to `/site-admin/report-bug`.
+
+Note that secrets and tokens are not automatically redacted. Please review the full settings export before sharing externally.
+
+
+## Experimental paginated search API
+
+To enable better programmatic consumption of search results, Sourcegraph 3.9 introduces the ability to consume an entire search result set via multiple paginated search requests. The results will be returned with a stable order. This [new paginated search API](https://docs.sourcegraph.com/api/graphql/search#sourcegraph-3-9-experimental-paginated-search) is experimental and is not yet ready for production use, but we are eager to hear feedback from early adopters as we work to further improve it. (If you're interested, see the [paginated search documentation](https://docs.sourcegraph.com/api/graphql/search#sourcegraph-3-9-experimental-paginated-search).)
```
Comment (slimsag) [inline blogposts/announcing-sourcegraph-3.9.md]:
```suggestion
To enable better programmatic consumption of search results, Sourcegraph 3.9 introduces the ability to consume an entire search result set via multiple paginated search requests. The results will be returned with a stable order. This [new paginated search API](https://docs.sourcegraph.com/api/graphql/search#sourcegraph-3-9-experimental-paginated-search) is experimental and is not yet ready for production use, but we are eager to hear feedback from early adopters as we work to further improve it. (If you're interested, see the [paginated search documentation](https://docs.sourcegraph.com/api/graphql/search#sourcegraph-3-9-experimental-paginated-search) and [architecture design document](https://docs.sourcegraph.com/dev/architecture/search-pagination))
```

**Comment by slimsag** [comment]
@sqs I see, my bad, I missed the first link to the API docs. I made a suggested fix which I think is more clear (linking to both at the end). Does that resolve the issue?

**Pull request merged**